src/utils/color.js

```diff
@@ -0,0 +1,19 @@
+export function componentToHex(c) {
+    var hex = c.toString(16);
+    return hex.length == 1 ? "0" + hex : hex;
+}
+
+export function rgbToHex([r, g, b]) {
+    return "#" + componentToHex(r) + componentToHex(g) + componentToHex(b);
+}
+
+// Unique color generation
+// https://stackoverflow.com/questions/15804149/rgb-color-permutation/15804183#15804183
+export function generateNexUniqueColor(i) {
```

Comment:
```suggestion
export function generateNextUniqueColor(i) {
```

Minor name change. Other than these small things, everything looks good

Reply:
Thank you for the feedback!